<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>yonetici</class>
 <widget class="QWidget" name="yonetici">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="giris">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>330</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color: rgb(230, 115, 0);</string>
   </property>
   <property name="text">
    <string>Giriş Yap</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>111</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Kullanıcı Adı:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Şifre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yon_ad">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>220</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yon_sifre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>270</y>
     <width>179</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-width:2px;
border-style:solid;
border-color: rgb(230, 115, 0);</string>
   </property>
   <property name="echoMode">
    <enum>QLineEdit::Password</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>0</y>
     <width>189</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:#E67300;</string>
   </property>
   <property name="text">
    <string>YÖNETİCİ GİRİŞİ</string>
   </property>
  </widget>
  <widget class="QLabel" name="yanlis">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>300</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="giris_2">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>133</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="arka_plan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>481</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>arka_plan</zorder>
  <zorder>giris</zorder>
  <zorder>label_2</zorder>
  <zorder>label_3</zorder>
  <zorder>yon_ad</zorder>
  <zorder>yon_sifre</zorder>
  <zorder>label</zorder>
  <zorder>yanlis</zorder>
  <zorder>giris_2</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
